runatlantis.io/docs/server-configuration.md

```diff
@@ -276,6 +276,13 @@ Values are chosen in this order:
   ```
   View help.
 
+* ### `--hide-prev-plan-comments`
+  ```bash
+  atlantis server --hide-prev-plan-comments
+  ```
+  Hide previous plan comments to declutter PRs. This is only supported in
+  GitHub currently.
```

Comment:
@lkysow I'm trying to figure out how to set a boolean flag like this via environment variable - before I go testing random settings (1, true, etc) maybe there could be some docs added to show how to do this? 

Maybe a more generic bit in the "environment variable" section of the config docs would handle it for any boolean flags?

Reply:
You can set it to the string `false` or `true`, e.g. `ATLANTIS_HIDE_PREV_PLAN_COMMENTS=true`. Yeah I think adding this to https://www.runatlantis.io/docs/server-configuration.html#environment-variables would be great. PRs welcome please!

Reply:
@lkysow opened a PR for the docs: https://github.com/runatlantis/atlantis/pull/1017